introduction.md

```diff
@@ -1,3 +1,7 @@
 This is an introduction.
 
 Ruby Path
+* Java
+* Ruby
+* Git
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?